guardian/facia-tool
# Fix Jackson vulnerabilities

This resolves high vulnerabilities in the Jackson libraries which are brought in transitively via the AWS SDK. 

The vulnerabilities have been resolved on the AWS-side (https://github.com/aws/aws-sdk-java/issues/2726), so we can stop overriding these libraries and bump the AWS SDK version instead.

## How to test

Check the app compiles and runs as expected.

The Snyk Github Action has been run manually on this branch and you can see the snapshot for this no longer contains the Jackson vulnerabilties: https://app.snyk.io/org/guardian/project/3bfd962f-25eb-4fca-9c02-322bdd8422a2 (the medium Jackson vulnerabilities are still there, but the high one has been resolved).
